pkg/maps/filter/sock_filter.go

```diff
@@ -0,0 +1,122 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package filter
```

Comment:
`filter` is very generic package name to use here. I'd suggest just using `socktermfilter` instead or something similar.

Reply:
Yeah, renamed this to `socktermfilter` in the latest push.